keyboards/handwired/3dfoxc/README.md

```diff
@@ -0,0 +1,20 @@
+# 3dfoxc
+
```

Comment:
Please use the default template, rename to be lower case, and remove all html tags.

https://github.com/qmk/qmk_firmware/blob/master/data/templates/avr/readme.md